main/tests/server/src/com/google/refine/expr/functions/strings/ParseUriTest.java

```diff
@@ -0,0 +1,48 @@
+
+package com.google.refine.expr.functions.strings;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.google.refine.RefineTest;
+import com.google.refine.expr.EvalError;
+import com.google.refine.util.ParsingUtilities;
+import org.slf4j.LoggerFactory;
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.util.HashMap;
+
+public class ParseUriTest extends RefineTest {
+
+    private String sampleUri;
+
+    @Override
+    @BeforeTest
+    public void init() {
+        logger = LoggerFactory.getLogger(this.getClass());
+        sampleUri = "https://www.openrefine.org:80/documentation#download";
```

Comment:
```suggestion
        sampleUri = "https://www.openrefine.org:80/documentation#download?format=xml&os=mac";
```
Could our example and test be more complex? More exhaustive?